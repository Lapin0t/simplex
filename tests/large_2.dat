40
40
0 0 0 0 159 98 30 0 0 0 0 -123 35 0 0 0 0 0 0 0 0 0 -65 0 0 0 0 120 0 0 0 0 0 -193 0 191 0 0 -83 -14
-63 -10638 -4123 1760 -21676 -3108 -2166 1249 -13829 6822 -15256 891 -25068 -38169 -31321 -15836 -8224 -10635 -10556 1865 3710 -4788 -9688 -692 -10913 -19166 -24542 5040 -7696 4194 -19480 -5191 16042 1648 -9921 20470 -17441 -10873 -17713 -23355
0 0 0 115 175 -29 -63 0 0 0 0 0 0 0 0 154 141 0 0 0 71 83 -169 0 83 0 0 -117 0 0 0 0 0 0 0 0 0 42 0 0
0 0 0 -188 0 0 0 131 0 192 133 114 189 0 0 0 57 0 0 0 0 0 0 0 173 0 -40 140 0 0 0 -23 0 42 0 0 0 0 0 0
0 122 0 0 0 0 0 -112 0 0 117 0 0 0 0 0 0 0 177 0 -30 0 0 167 0 77 0 0 0 -15 0 96 -86 0 0 0 89 0 82 0
0 0 -170 145 0 0 -144 0 -34 0 0 0 0 0 0 27 0 117 126 0 -166 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 -21 -12 -195
0 0 100 145 0 0 0 -83 0 0 0 0 0 152 0 0 0 0 0 -29 188 -197 0 -83 0 148 0 -162 0 0 0 0 0 0 0 0 -48 0 0 0
21 93 0 0 16 0 0 0 0 27 0 0 0 0 0 -30 0 0 0 0 0 0 69 0 -172 0 106 0 0 0 -194 0 0 0 79 -123 0 0 0 0
0 -47 0 0 0 -16 0 0 180 0 0 0 -6 0 0 0 0 194 0 0 0 0 0 0 0 68 100 0 -6 0 0 88 0 0 0 0 0 0 0 1
0 -73 0 -87 0 186 -60 -16 0 109 0 0 22 -111 0 0 0 0 0 0 0 41 0 0 199 0 -112 0 83 0 0 0 0 0 0 0 0 0 0 0
0 -90 0 -158 182 0 0 0 199 -145 0 0 62 187 0 0 0 -138 -96 0 0 0 0 0 0 0 42 0 -83 0 0 -145 0 45 0 10 0 0 0 0
-4 0 0 0 0 190 -66 -27 0 8 0 0 0 0 0 0 0 0 0 0 0 0 0 0 -165 -133 0 76 0 -107 58 0 0 0 -193 42 22 -164 0 78
0 0 0 103 0 178 0 0 0 0 -138 193 0 -7 -71 0 0 0 0 0 0 0 106 165 0 0 61 128 0 -6 0 -37 90 0 0 0 0 0 0 0
117 0 83 0 0 -135 0 0 0 0 0 -117 0 -65 0 0 0 0 0 0 0 0 0 165 0 0 0 0 0 0 -72 -103 0 0 0 -13 0 0 0 0
0 0 0 37 112 0 0 -50 0 -182 0 0 0 0 0 0 0 0 0 0 0 0 137 0 0 8 0 0 190 0 0 0 0 0 0 0 0 0 0 0
186 0 -101 -50 0 -170 11 -98 0 0 -160 68 0 -5 37 0 0 0 0 0 -42 -137 -75 67 -3 0 0 0 0 15 0 36 0 137 0 0 172 0 0 0
190 0 -116 -195 0 0 0 -146 -13 0 0 0 0 0 0 63 19 -5 61 0 199 0 0 0 0 0 0 0 24 0 0 0 139 0 -71 0 0 0 0 0
20 0 0 47 0 0 0 0 0 -108 0 0 0 0 9 0 0 0 0 0 0 0 169 -151 0 0 0 -137 0 180 0 -30 184 0 0 0 0 0 174 0
0 0 -42 0 0 0 0 0 0 0 0 0 0 0 0 0 -182 0 88 111 0 0 0 0 0 0 0 0 75 0 0 105 -104 0 0 0 -70 0 7 0
0 -7 -90 0 0 0 0 -107 39 12 25 0 0 0 0 0 0 0 -55 -26 -63 0 0 0 0 0 0 -65 0 0 0 0 0 0 0 0 0 0 100 0
91 0 0 -5 73 0 0 0 -171 0 0 0 0 -171 0 0 0 0 -195 0 0 0 196 0 49 107 183 -13 0 0 0 0 0 170 14 0 0 0 0 0
0 0 0 0 0 -106 0 0 0 0 0 -78 -143 0 0 0 68 0 -163 67 0 0 -198 0 -15 0 156 9 0 0 0 0 0 0 0 0 0 0 -6 0
0 0 -12 0 0 0 0 0 0 0 104 -45 0 0 0 -5 0 0 0 -44 -59 0 -156 0 -67 0 -85 0 0 0 0 0 34 0 0 -13 96 0 0 0
62 0 0 0 -12 0 172 0 78 0 0 0 -71 0 63 0 0 67 136 91 -128 110 196 0 -162 0 156 77 0 58 0 0 0 0 55 0 0 0 0 17
0 0 0 0 0 -47 0 0 -200 0 0 0 0 0 0 114 0 0 0 153 0 -178 114 0 83 -137 0 0 12 0 0 90 0 0 0 0 -72 0 195 -107
0 55 0 0 162 131 0 140 0 0 0 0 0 -4 -30 -5 0 0 0 12 -92 97 0 0 0 0 33 0 81 70 0 -166 0 21 0 0 0 0 -135 199
0 0 0 -90 0 0 -40 0 0 13 28 131 -185 0 0 118 0 0 0 -57 0 -96 195 0 0 0 -74 0 0 0 0 0 0 0 0 0 -64 48 0 0
0 -188 0 0 0 0 0 0 0 33 0 162 0 0 0 -137 0 94 0 0 0 0 0 43 0 0 111 0 44 0 0 47 -10 0 180 198 0 109 0 0
176 -143 128 0 0 0 0 0 0 0 -39 0 0 0 -171 -139 0 71 0 -56 0 -86 0 0 -174 0 -36 0 0 0 -99 0 0 90 0 0 0 0 -25 0
0 0 0 0 0 183 0 0 0 0 34 0 0 0 0 -124 25 -95 0 0 0 0 0 0 105 0 0 119 0 59 0 -134 174 -180 -79 0 3 0 0 0
87 0 0 0 0 0 0 -114 0 19 0 0 69 0 0 0 0 0 0 0 0 0 0 0 78 0 -176 0 0 39 -5 0 0 -185 0 0 0 -132 0 0
0 0 0 0 0 0 0 0 198 0 -82 0 0 0 0 0 -179 120 0 0 -119 0 -168 0 -129 171 0 199 -92 130 -21 165 0 0 0 0 0 0 0 152
0 0 0 152 0 34 0 131 0 0 67 0 0 186 0 159 0 0 0 0 0 0 0 0 152 -83 0 0 0 150 0 0 -114 108 0 -3 94 72 0 0
0 0 105 0 0 0 0 7 0 0 0 0 0 64 0 0 72 0 0 27 0 -195 0 82 144 0 -146 0 0 0 -123 0 0 0 19 -82 -83 0 0 0
0 75 0 0 0 -101 0 170 0 0 0 -142 -119 0 0 38 196 38 0 0 -73 0 0 0 0 0 0 0 0 -97 0 27 0 -152 -165 0 0 0 0 0
0 0 0 0 0 0 0 0 11 0 179 0 0 0 0 0 -62 25 49 0 0 -40 -133 35 0 0 0 -178 0 0 0 -53 169 0 0 81 0 0 0 -186
0 -115 0 0 0 120 0 0 -190 0 0 0 0 -21 0 0 0 0 0 0 0 49 0 0 0 0 0 0 0 0 -163 0 0 0 0 0 -61 0 165 0
-139 105 0 0 0 185 -102 0 -157 0 195 0 0 0 0 0 -174 0 0 0 60 0 -31 0 0 0 0 -138 0 0 0 0 0 0 0 126 0 0 0 0
52 0 0 0 0 -188 0 0 0 90 0 0 0 0 0 -40 0 0 -55 0 0 0 0 0 0 0 0 0 0 0 0 -83 0 -67 0 -12 0 -157 134 0
0 0 0 -37 0 15 0 0 86 0 0 131 133 0 54 0 112 -52 0 172 163 57 0 0 0 0 -187 86 0 0 0 0 -193 0 0 0 155 190 0 149
-89 0 0 -155 0 0 -16 0 0 69 0 111 0 0 0 0 0 0 0 0 0 -175 0 0 0 189 0 0 115 0 -84 84 -154 -71 0 0 0 130 0 0
0 0 0 0 0 151 0 162 0 0 37 0 0 -21 0 -101 0 -180 0 -106 0 0 0 0 139 0 -162 0 174 0 0 0 0 108 0 0 0 46 0 0
