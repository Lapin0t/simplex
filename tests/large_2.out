INFO: maximize:
INFO:                                   +159x_4  +98x_5  +30x_6                                  -123x_11  +35x_12                                                                                   -65x_22                                     +120x_27                                              -193x_33          +191x_35                    -83x_38  -14x_39
INFO: subject to:
INFO:                           +115x_3 +175x_4  -29x_5  -63x_6                                                                      +154x_15 +141x_16                             +71x_20  +83x_21 -169x_22           +83x_24                   -117x_27                                                                                   +42x_37                   <= -63
INFO:                           -188x_3                         +131x_7         +192x_9 +133x_10 +114x_11 +189x_12                             +57x_16                                                                +173x_24           -40x_26 +140x_27                             -23x_31           +42x_33                                                       <= -10638
INFO:           +122x_1                                         -112x_7                 +117x_10                                                                +177x_18           -30x_20                   +167x_23           +77x_25                             -15x_29           +96x_31  -86x_32                             +89x_36           +82x_38          <= -4123
INFO:                   -170x_2 +145x_3                 -144x_6          -34x_8                                                       +27x_15          +117x_17 +126x_18          -166x_20                                                                                                                                                  -21x_37  -12x_38 -195x_39 <= 1760
INFO:                   +100x_2 +145x_3                          -83x_7                                            +152x_13                                               -29x_19 +188x_20 -197x_21           -83x_23          +148x_25          -162x_27                                                                          -48x_36                            <= -21676
INFO:    +21x_0  +93x_1                  +16x_4                                  +27x_9                                               -30x_15                                                        +69x_22          -172x_24          +106x_26                            -194x_30                             +79x_34 -123x_35                                     <= -3108
INFO:            -47x_1                          -16x_5                 +180x_8                             -6x_12                                     +194x_17                                                                 +68x_25 +100x_26            -6x_28                    +88x_31                                                                   +x_39 <= -2166
INFO:            -73x_1          -87x_3         +186x_5  -60x_6  -16x_7         +109x_9                    +22x_12 -111x_13                                                                 +41x_21                   +199x_24          -112x_26           +83x_28                                                                                                    <= 1249
INFO:            -90x_1         -158x_3 +182x_4                         +199x_8 -145x_9                    +62x_12 +187x_13                            -138x_17  -96x_18                                                                 +42x_26           -83x_28                   -145x_31           +45x_33           +10x_35                                     <= -13829
INFO:     -4x_0                                 +190x_5  -66x_6  -27x_7           +8x_9                                                                                                                               -165x_24 -133x_25           +76x_27          -107x_29  +58x_30                            -193x_34  +42x_35  +22x_36 -164x_37           +78x_39 <= 6822
INFO:                           +103x_3         +178x_5                                 -138x_10 +193x_11            -7x_13  -71x_14                                                                +106x_22 +165x_23                    +61x_26 +128x_27            -6x_29           -37x_31  +90x_32                                                                <= -15256
INFO:   +117x_0          +83x_2                 -135x_5                                          -117x_11           -65x_13                                                                                  +165x_23                                                        -72x_30 -103x_31                             -13x_35                                     <= 891
INFO:                            +37x_3 +112x_4                  -50x_7         -182x_9                                                                                                             +137x_22                     +8x_25                   +190x_28                                                                                                    <= -25068
INFO:   +186x_0         -101x_2  -50x_3         -170x_5  +11x_6  -98x_7                 -160x_10  +68x_11            -5x_13  +37x_14                                               -42x_20 -137x_21  -75x_22  +67x_23   -3x_24                                      +15x_29           +36x_31          +137x_33                   +172x_36                            <= -38169
INFO:   +190x_0         -116x_2 -195x_3                         -146x_7  -13x_8                                                       +63x_15  +19x_16   -5x_17  +61x_18          +199x_20                                                                 +24x_28                            +139x_32           -71x_34                                              <= -31321
INFO:    +20x_0                  +47x_3                                         -108x_9                                       +9x_14                                                                +169x_22 -151x_23                            -137x_27          +180x_29           -30x_31 +184x_32                                              +174x_38          <= -15836
INFO:                    -42x_2                                                                                                               -182x_16           +88x_18 +111x_19                                                                          +75x_28                   +105x_31 -104x_32                             -70x_36            +7x_38          <= -8224
INFO:             -7x_1  -90x_2                                 -107x_7  +39x_8  +12x_9  +25x_10                                                                 -55x_18  -26x_19  -63x_20                                                        -65x_27                                                                                           +100x_38          <= -10635
INFO:    +91x_0                   -5x_3  +73x_4                         -171x_8                                    -171x_13                                     -195x_18                            +196x_22           +49x_24 +107x_25 +183x_26  -13x_27                                              +170x_33  +14x_34                                              <= -10556
INFO:                                           -106x_5                                           -78x_11 -143x_12                             +68x_16          -163x_18  +67x_19                   -198x_22           -15x_24          +156x_26   +9x_27                                                                                             -6x_38          <= 1865
INFO:                    -12x_2                                                         +104x_10  -45x_11                              -5x_15                             -44x_19  -59x_20          -156x_22           -67x_24           -85x_26                                               +34x_32                    -13x_35  +96x_36                            <= 3710
INFO:    +62x_0                          -12x_4         +172x_6          +78x_8                            -71x_12           +63x_14                    +67x_17 +136x_18  +91x_19 -128x_20 +110x_21 +196x_22          -162x_24          +156x_26  +77x_27           +58x_29                                      +55x_34                                      +17x_39 <= -4788
INFO:                                            -47x_5                 -200x_8                                                      +114x_15                            +153x_19          -178x_21 +114x_22           +83x_24 -137x_25                    +12x_28                    +90x_31                                      -72x_36          +195x_38 -107x_39 <= -9688
INFO:            +55x_1                 +162x_4 +131x_5         +140x_7                                              -4x_13  -30x_14   -5x_15                             +12x_19  -92x_20  +97x_21                                      +33x_26           +81x_28  +70x_29          -166x_31           +21x_33                                     -135x_38 +199x_39 <= -692
INFO:                            -90x_3                  -40x_6                  +13x_9  +28x_10 +131x_11 -185x_12                   +118x_15                             -57x_19           -96x_21 +195x_22                             -74x_26                                                                                   -64x_36  +48x_37                   <= -10913
INFO:           -188x_1                                                          +33x_9          +162x_11                            -137x_15           +94x_17                                               +43x_23                   +111x_26           +44x_28                    +47x_31  -10x_32          +180x_34 +198x_35          +109x_37                   <= -19166
INFO:   +176x_0 -143x_1 +128x_2                                                          -39x_10                            -171x_14 -139x_15           +71x_17           -56x_19           -86x_21                   -174x_24           -36x_26                             -99x_30                    +90x_33                                      -25x_38          <= -24542
INFO:                                           +183x_5                                  +34x_10                                     -124x_15  +25x_16  -95x_17                                                       +105x_24                   +119x_27           +59x_29          -134x_31 +174x_32 -180x_33  -79x_34            +3x_36                            <= 5040
INFO:    +87x_0                                                 -114x_7          +19x_9                    +69x_12                                                                                                     +78x_24          -176x_26                    +39x_29   -5x_30                   -185x_33                            -132x_37                   <= -7696
INFO:                                                                   +198x_8          -82x_10                                              -179x_16 +120x_17                   -119x_20          -168x_22          -129x_24 +171x_25          +199x_27  -92x_28 +130x_29  -21x_30 +165x_31                                                                +152x_39 <= 4194
INFO:                           +152x_3          +34x_5         +131x_7                  +67x_10                   +186x_13          +159x_15                                                                         +152x_24  -83x_25                            +150x_29                   -114x_32 +108x_33            -3x_35  +94x_36  +72x_37                   <= -19480
INFO:                   +105x_2                                   +7x_7                                             +64x_13                    +72x_16                    +27x_19          -195x_21           +82x_23 +144x_24          -146x_26                            -123x_30                             +19x_34  -82x_35  -83x_36                            <= -5191
INFO:            +75x_1                         -101x_5         +170x_7                          -142x_11 -119x_12                    +38x_15 +196x_16  +38x_17                    -73x_20                                                                          -97x_29           +27x_31          -152x_33 -165x_34                                              <= 16042
INFO:                                                                    +11x_8         +179x_10                                               -62x_16  +25x_17  +49x_18                    -40x_21 -133x_22  +35x_23                            -178x_27                             -53x_31 +169x_32                    +81x_35                            -186x_39 <= 1648
INFO:           -115x_1                         +120x_5                 -190x_8                                     -21x_13                                                                 +49x_21                                                                         -163x_30                                               -61x_36          +165x_38          <= -9921
INFO:   -139x_0 +105x_1                         +185x_5 -102x_6         -157x_8         +195x_10                                              -174x_16                             +60x_20           -31x_22                                     -138x_27                                                                +126x_35                                     <= 20470
INFO:    +52x_0                                 -188x_5                          +90x_9                                               -40x_15                    -55x_18                                                                                                              -83x_31           -67x_33           -12x_35          -157x_37 +134x_38          <= -17441
INFO:                            -37x_3          +15x_5                  +86x_8                  +131x_11 +133x_12           +54x_14          +112x_16  -52x_17          +172x_19 +163x_20  +57x_21                                     -187x_26  +86x_27                                     -193x_32                            +155x_36 +190x_37          +149x_39 <= -10873
INFO:    -89x_0                 -155x_3                  -16x_6                  +69x_9          +111x_11                                                                                  -175x_21                            +189x_25                   +115x_28           -84x_30  +84x_31 -154x_32  -71x_33                            +130x_37                   <= -17713
INFO:                                           +151x_5         +162x_7                  +37x_10                    -21x_13          -101x_15          -180x_17          -106x_19                                     +139x_24          -162x_26          +174x_28                                     +108x_33                             +46x_37                   <= -23355
INFO:   x_0, x_1, x_2, x_3, x_4, x_5, x_6, x_7, x_8, x_9, x_10, x_11, x_12, x_13, x_14, x_15, x_16, x_17, x_18, x_19, x_20, x_21, x_22, x_23, x_24, x_25, x_26, x_27, x_28, x_29, x_30, x_31, x_32, x_33, x_34, x_35, x_36, x_37, x_38, x_39, x_40, x_41, x_42, x_43, x_44, x_45, x_46, x_47, x_48, x_49, x_50, x_51, x_52, x_53, x_54, x_55, x_56, x_57, x_58, x_59, x_60, x_61, x_62, x_63, x_64, x_65, x_66, x_67, x_68, x_69, x_70, x_71, x_72, x_73, x_74, x_75, x_76, x_77, x_78, x_79 >= 0
INFO: number of variables: 40
INFO: number of constraints: 40
INFO: PHASE 1 (29 bad constraints)
INFO: PHASE 2 (initial base: [ 0  3  4 71 56 31  1 47 57 45  5 67  6 73  7  8  9 48 10 72 60 11 30 12 43
  2 74 78 34 13 14 21 15 32 51 75 16 17 26 18])
INFO: number of pivot steps (both phases): 198
INFO: rule used: greedy
BOUNDED
OPTIMAL VALUE:  -5937473920073530327874878038097/445353446175090344049084
SOLUTION:  x_0 = 0, x_1 = -1657961280087094454461038974/37112787181257528670757, x_2 = -870109582391348091455463569/37112787181257528670757, x_3 = 0, x_4 = -104081107125157012978627278757/668030169262635516073626, x_5 = 1765237306773636204116816800/37112787181257528670757, x_6 = 0, x_7 = 0, x_8 = 0, x_9 = -32268443859952433760625547674/334015084631317758036813, x_10 = 0, x_11 = 0, x_12 = 0, x_13 = 0, x_14 = -1758112640981000903985438201/37112787181257528670757, x_15 = 0, x_16 = 1919847035284543439957572217/37112787181257528670757, x_17 = 0, x_18 = 0, x_19 = 0, x_20 = 0, x_21 = 99138822428007898697445042/37112787181257528670757, x_22 = 0, x_23 = 0, x_24 = 0, x_25 = -731776022834556822072067980/37112787181257528670757, x_26 = 0, x_27 = -1321232721370529855951492361/37112787181257528670757, x_28 = 0, x_29 = 0, x_30 = 27193584158997645199472105974031/6680301692626355160736260, x_31 = 0, x_32 = -6010814276914838929141545539/74225574362515057341514, x_33 = -8500482696374576622432445515/148451148725030114683028, x_34 = 396076934055373351908804380458/6123609884907492230674905, x_35 = 0, x_36 = 0, x_37 = -13032323133438610776991768151/148451148725030114683028, x_38 = 0, x_39 = 0
